examples/reference/panes/HoloViews.ipynb

```diff
@@ -119,18 +207,205 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "widgets = hv_panel[1]\n",
+    "plot = df.hvplot.scatter(x=\"group\", y=\"value\")\n",
+    "pn.pane.HoloViews(plot, backend='matplotlib', sizing_mode=\"stretch_width\", height=300)"
```

Comment:
I would like to add an example where the matplotlib plot stretches responsively. As that is what I concretely would use in applications and when I try to communicate the message that hvPlot/ HoloViews is pretty flexible because you can replace bokeh with matplotlib or plotly easily.

How do I do that?

Reply:
You can now set `format='svg'` and then get responsive plots.